db/data_migration/20240704092457_merge_roles_for_minister_for_disabled_people_health_work.rb

```diff
@@ -0,0 +1,52 @@
+role_to_merge = Role.find_by(id: 2377)
+role_to_keep = Role.find_by(id: 4494)
+
+unless role_to_merge && role_to_keep
+  puts "Roles not found - exiting."
+  return
+end
+
+puts "merging #{role_to_merge.slug} into #{role_to_keep.slug}"
+
+puts "Validating has no edition_role association..."
+puts "count: ", EditionRole.where(role_id: role_to_merge.id).count
+
+puts "Validating has no historical_account_roles association..."
+puts "count: ", HistoricalAccountRole.where(role_id: role_to_merge.id).count
+
+puts "Validating has no worldwide_organisation_roles association..."
+puts "count: ", WorldwideOrganisationRole.where(role_id: role_to_merge.id).count
```

Comment:
I think the three "validating" lines don't stop us from doing the edition role appointment and organisation updates - we have a pretty high degree of confidence it will be 0. 
These lines are just more like let's be 100% sure it is 0, and if it is not 0, then we'll have to write additional migrations to deal with them.